src/test/kotlin/no/nav/syfo/tilgang/TilgangServiceSpek.kt

```diff
@@ -89,27 +99,22 @@ class TilgangServiceSpek : Spek({
                 val callId = "123"
                 every { redisStore.getObject<Tilgang?>(any()) } returns null
                 coEvery { axsysClient.getEnheter(validToken, callId) } returns listOf(veiledersEnhet)
+                coEvery { graphApiClient.hasAccess(any(), any(), any()) } returns true
```

Comment:
Hvorfor `any()` på alle her? Fordi det bare er for å få testen til å kjøre, uten at vi egentlig tester at SYFO-tilgang må være på plass?

Reply:
Dette er set up av testen, så her bør man jo strengt tatt gjøre det så enkelt og vedlikeholdsfritt som mulig. Så jeg endrer i linja over også jeg..